src/Chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -1,7 +1,7 @@
 $ErrorActionPreference = 'Stop';
 $packageName = 'carnac'
-$url = 'Download Url Here'
-$zipFileHash = 'Zip File Hash Here'
+$url = 'https://github.com/Code52/carnac/releases/download/1.0.0/carnac.1.0.0.zip'
```

Comment:
We swap this out at build time - can we swap this back to the broken values instead of accidentally publishing old versions?